### PR TITLE
add various php modules, update kudu runtime

### PR DESCRIPTION
**Pull request by patricklee2**
- fixes: https://github.com/projectkudu/kudu/issues/2702
- update kudu runtime
- https://github.com/Azure-App-Service/php/pull/29

### CODE REVIEW

**Comment by patricklee2** [comment]
```# php -m
[PHP Modules]
bcmath
calendar
Core
ctype
curl
date
dom
exif
fileinfo
filter
ftp
gd
gettext
hash
iconv
imap
intl
json
libxml
mbstring
mysqli
mysqlnd
odbc
openssl
pcntl
pcre
PDO
pdo_mysql
PDO_ODBC
pdo_pgsql
pdo_sqlite
pdo_sqlsrv
pgsql
Phar
posix
readline
Reflection
session
shmop
SimpleXML
soap
sockets
sodium
SPL
sqlite3
sqlsrv
standard
sysvmsg
sysvsem
sysvshm
tidy
tokenizer
wddx
xml
xmlreader
xmlrpc
xmlwriter
xsl
Zend OPcache
zip
zlib

[Zend Modules]
Zend OPcache
```

**Comment by patricklee2** [comment]
com_dotnet, wincache - Windows Only Extensions
ereg, mcrypt, mhash, mysql - deprecated

File: kudu/Dockerfile [hunk 1]
```diff
@@ -308,7 +308,17 @@ RUN mkdir /var/nuget \
...
+  && wget -O /etc/apt/trusted.gpg.d/php.gpg https://packages.sury.org/php/apt.gpg \
+  && sh -c 'echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" > /etc/apt/sources.list.d/php.list' \ 
+  && apt-get update \
+  && apt-get install -y --no-install-recommends php5-cli php5.6-zip php5.6-mbstring php5.6-xml php5-mysql php5-pgsql php5-curl php5-json php5-gd php5-gmp php5-imagick php5-imap php5-intl php5-ldap php5.6-mbstring php5-odbc php5-readline php5.6-SimpleXML php5.6-soap php5-sqlite php5-tidy php5.6-wddx php5-xmlrpc php5.6-xmlwriter php5-xsl
+
+# mcrypt
+RUN apt-get install -y --no-install-recommends php5-mcrypt \
+  && php5enmod mcrypt \
+  && apt-get install -y apache2 \
+  && service apache2 restart
```
Comment (rramachand21) [inline kudu/Dockerfile]: can we restart apache just once after installing all the necessary modules?

File: kudu/Dockerfile [hunk 2]
```diff
@@ -356,16 +380,16 @@ RUN curl -fL -o /usr/local/go/bin/dep $DEP_DOWNLOAD_URL \
 COPY kudu.conf /tmp
 COPY apache2.conf /tmp
 RUN npm install -g kudusync \
-  && wget -O /tmp/Kudu.zip https://nildev.blob.core.windows.net/kudu/Kudu.71.10312.3298.zip \
+  && wget -O /tmp/Kudu.zip https://patlekudu.blob.core.windows.net/kudu/Kudu.75.10626.3456.zip \
```
Comment (rramachand21) [inline kudu/Dockerfile]: Please use this URL https://wawsinfraprodbay063.blob.core.windows.net/kudu/Kudu.75.10626.3456.zip

**Pull request merged**